docs/current_docs/729312-install.mdx

```diff
@@ -8,5 +8,5 @@ displayed_sidebar: 'current'
 Dagger SDKs are available for multiple programming languages. Refer to the links below for installation instructions for your language of choice.
 
 - [Install the Dagger Go SDK](./sdk/go/371491-install.mdx)
-- [Install the Dagger Node.js SDK](./sdk/nodejs/835948-install.mdx)
+- [Install the Dagger Typescript SDK](./sdk/typescript/835948-install.mdx)
```

Comment:
Minor but important nit which affects this entire diff: we should use `TypeScript` instead of `Typescript` as this is the official capitalization, see https://www.typescriptlang.org/

Reply:
Sounds good to me, I'll update that

Reply:
Done in https://github.com/dagger/dagger/pull/6373/commits/9a1860e559a558835fae8935111caea0b1c9e622